src/OrderFormAcceptanceTests.Steps/Steps/Hooks.cs

```diff
@@ -13,7 +13,8 @@ public Hooks(UITest test, ScenarioContext context) : base(test, context)
         [AfterScenario]
         public void AfterScenario()
         {
-            Test.Driver.Quit();
+            Test.Driver?.Close();
```

Comment:
what does the ? do? and why do you need close and quit?

Reply:
the `?` is a null check, to make sure it doesn't just bomb out if the driver is null for some reason. I have changed this slightly in the next commit to not have the close